tests/lifecycle/driver_test.go

```diff
@@ -46,7 +46,7 @@ func TestServices(t *testing.T) {
 			t.Run(tc.getName(), func(t *testing.T) {
 				// Run subtests in parallel!
 				t.Parallel()
-				err := tc.execute(resourceGroup)
+				err := tc.execute(t, resourceGroup)
```

Comment:
This can be a little bit of a problem, iirc. If you pass `t` around to "test helpers" and make assertions in those helpers, I do not think that the output from the test run will give correct attribution to the real source of the error. (There might be some ways around that that have emerged in the last few months. I'll research that a bit.) It might be safer here to do as we were before-- treat `tc.execute()` and everything _it_ calls as "normal" functions that return errors instead of directly failing the tests using `t`.